README.md

```diff
@@ -21,43 +21,52 @@ The goal is to provide a smooth out-of-the-box experience for simple use cases,
 allow swapping in more powerful backends, like Mesos, for large scale production
 deployments.
 
-## Installation and documentation
+## Installation for Swarm Users
 
-Full documentation [is available here](http://docs.docker.com/swarm/).
+For installing swarm for using in your environment, use the Docker Swarm
+documentation on [docs.docker.com](http://docs.docker.com/swarm/).
 
-## Development installation
+## Installation for Swarm Developers
 
-You can download and install from source instead of using the Docker
-image. Ensure you have golang, godep and the git client installed.
+Developers should always download and install from source rather than
```

Comment:
@JamesYongQiaoWang Thank you for the contribution.

Remove "always" from this sentence.